MeshLib/MeshSearch/NodeSearch.cpp

```diff
@@ -13,7 +13,7 @@
 #include <memory>
 #include <set>
 
-#include <logog/include/logog.hpp>
+#include "BaseLib/Logging.h"
```

Comment:
As far as I can see there isn't any logging output in this file - maybe the include can be removed.

Reply:
Yeah. This is another PR for header includes cleanup.